SU-HSDO/stanford_mrc
# MRCD8-251 Fixed font and image 404 errors

# READY FOR REVIEW

# Summary
- Fixed 404 errors with asset files

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout this branch
2. verify paths match those configured in the css files.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
